src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ballOne = new Lottery();
+        Lottery ballTwo = new Lottery();
+        Lottery ballThree = new Lottery();
+        Lottery[] ballWin = {ballOne, ballTwo, ballThree};
+        for (Lottery ball: ballWin) {
+            System.out.println(ball.getRandomBall());
+        }
     }
 }
```

Comment:
you need only one lottery instance, remove the array (array naming is weird, what does Win stand for?). Don't name lottery a ball, you have a separate Ball model. This is confusing. Ball is what's returned by getRandomBall

Reply:
I thought that it was necessary to create three balls separately, so I called them "winning balls"